xml/System.Security.Cryptography.X509Certificates/X509CertificateCollection.xml

```diff
@@ -975,9 +975,9 @@
         <Parameter Name="value" Type="System.Object" Index="0" FrameworkAlternate="netcore-1.0;netcore-1.1;netstandard-1.3;netstandard-1.4;netstandard-1.6" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The object to locate in the collection.</param>
+        <summary>Searches for the specified object and returns the zero-based index of the first occurrence within the collection.</summary>
+        <returns>The zero-based index of the first occurrence of <paramref name="value"/> within the collection, if found; otherwise -1.</returns>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
         <remarks>
           <format type="text/markdown"><![CDATA[

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Security.Cryptography.X509Certificates.X509CertificateCollection> instance is cast to an <xref:System.Collections.IList> interface.  

           ]]></format>
         </remarks>
```